ordered_set.pyi

```diff
@@ -0,0 +1,41 @@
+from typing import Any, MutableSet, TypeVar, Sequence, Union, Tuple, List, Iterator, Dict, Set, Iterable
+
+
+def is_iterable(obj: Any) -> bool: ...
+
+
+T = TypeVar('T')
+
+
+class OrderedSet(MutableSet[T], Sequence[T]):
+    def __init__(self, iterable: Iterable[T] = None) -> None:
+        self.map: Dict[T, int] = {}
+        self.items: List[T] = []
+
+    def __len__(self) -> int: ...
+    def __getitem__(self, index: Union[int, slice, Sequence[T]]) -> OrderedSet[T]: ...
```

Comment:
Shouldn't the signature of fancy indexing be `Sequence[int]` instead of `Sequence[T]`?

Reply:
Agreed. My mistake. Fixed it.